FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs

```diff
@@ -263,6 +263,41 @@ public async Task<FirebaseToken> VerifyIdTokenAsync(string idToken)
                 .ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Creates a new user account with the attributes contained in the specified <see cref="UserRecordArgs"/>.
+        /// </summary>
+        /// <param name="args">Attributes that will be added to the new user account.</param>
```

Comment:
Just so you don't think I'm being arbitrary about "will" -- I think it helpfully disambiguates here. If we said "attributes of" or "attributes for," it might suggest that the attributes already exist for the account.  So "will be" works best here :)